README.md

```diff
@@ -1,4 +1,4 @@
-# ember-m3 [![Build Status](https://secure.travis-ci.org/hjdivad/ember-m3.svg?branch=master)](http://travis-ci.org/hjdivad/ember-m3)
+# Ember-m3 [![Build Status](https://secure.travis-ci.org/hjdivad/ember-m3.svg?branch=master)](http://travis-ci.org/hjdivad/ember-m3)
```

Comment:
I think the only one dislike is what is here ATM (`Ember-m3`). 

IMHO, if we are referring to "the package" we should always use the package's actual name on NPM (which is `ember-m3`).  I personally would be happy to _only_ use `ember-m3`, but could also see where folks might prefer to use `Ember M3` when discussing things in prose.

Feel free to completely disregard this entire comment though 😝